hbase-server/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupInfoManagerImpl.java

```diff
@@ -1015,17 +1016,17 @@ private void moveServerRegionsFromGroup(Set<Address> movedServers, Set<Address>
               assignmentFutures.add(Pair.newPair(region, future));
             } catch (IOException ioe) {
               failedRegions.add(region.getRegionNameAsString());
-              LOG.debug("Move region {} from group failed, will retry, current retry time is {}",
-                region.getShortNameToLog(), retry, ioe);
+              LOG.debug("Move region {} from server {} failed, will retry, current retry time is {}",
```

Comment:
nit: 'server' in the log message is redundant; we are always moving between servers?